src/main/java/controllers/CommandLineInterface.java

```diff
@@ -1,7 +1,10 @@
 package controllers;
```

Comment:
double check spotless, some spacing is a lil sus

Reply:
no!